src/python/pants/backend/javascript/goals/test_integration_test.py

```diff
@@ -32,13 +32,14 @@
 ATTEMPTS_DEFAULT_OPTION = 2
 
 
-@pytest.fixture(params=["npm", "pnpm", "yarn"])
-def package_manager(request) -> str:
-    return cast(str, request.param)
+@pytest.fixture(params=[("npm", "10.8.1"), ("pnpm", "8.15.8"), ("yarn", "1.22.22")])
```

Comment:
I'm not sure if we should be pinning package manager versions for the integration tests - if there was a future breaking change in the default package manager versions, then we wouldn't catch it here? 

Perhaps this could reference the default package manager versions set in `nodejs.py`?

Reply:
Good point - I've removed all explicit setting of `packageManager` inside `package.json` and am instead using the RuleRunner to set the Node package manager. So far seems to work well!